### ISSUE TITLE
[ACT:afw4f7] Color contrast check should be limited to human-readable text

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1205]
Need a team discussion on exactly what counts as human-readable text

see pass example 7 [ACT afw4f7](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/afw4f7/proposed/).

**Comment by ErickRenteria** [comment]
Pushed
